databases/week2/meal_application.sql

```diff
@@ -0,0 +1,154 @@
+-- Part 4: Meal application
+
+-- create the database
+create database meal_share;
+
+-- create model
+use meal_share;
+
+create table Meal(
+	id INt auto_increment,
+	title varchar(30) NOT NULL,
+	description text, 
+	location varchar(50),
+	serving_time datetime,
+	max_reservation INT,
+    price decimal NOT NULL,
+    created_date datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    primary key (id)
+    );
+    
+create table Reservation (
+	id INT auto_increment,
+    number_of_guests INT,
+    meal_id INT,
+    created_date datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	primary key(id),
+    foreign key (meal_id) references Meal(id)
+    );
+    
+create table Review(
+	id INT auto_increment,
+    title varchar(30) NOT NULL,
+    description text,
+    meal_id int,
+    stars int,
+    created_date datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    primary key(id),
+    foreign key(meal_id) references Meal(id)
+    )    
+
+
+
+--Queries
+--Meal
+
+--Get all meals
+select * from Meal;
+--Add a new meal
+insert into Meal 
+values(default, ' Zereshk Polo','Barberry Rice','Copenhagenmm',null,null,5,null)
+-- Get a meal with any id, fx 1
+select * from Meal where Meal.id = 2;
+--Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
```

Comment:
You have just inserted one row, and you are querying with `where Meal.id = 2`. Why is that?